src/no-random-v4-uuid.ts

```diff
@@ -11,78 +11,108 @@ import getDocumentationUrl from './get-documentation-url.ts';
 
 export const ruleId = 'no-random-v4-uuid';
 const NO_RANDOM_V4_UUID = 'NO_RANDOM_V4_UUID';
+const NO_UUID_MODULE_FOR_V4 = 'NO_UUID_MODULE_FOR_V4';
 
 const createRule = ESLintUtils.RuleCreator((name) => getDocumentationUrl(name));
 
-// process the import declaration to get the alias for uuid.v4 and uuid.
 const processImportDeclaration = (
   node: TSESTree.ImportDeclaration,
-  uuid4Alias: string | undefined,
-  uuidDefaultAlias: string | undefined,
+  aliases: { uuid4Alias?: string; uuidDefaultAlias?: string; cryptoRandomUUIDAlias?: string },
 ) => {
-  let updatedUuid4Alias = uuid4Alias;
-  let updatedUuidDefaultAlias = uuidDefaultAlias;
   node.specifiers.forEach((specifier) => {
-    switch (specifier.type) {
-      case AST_NODE_TYPES.ImportSpecifier:
-        if (specifier.imported.type === AST_NODE_TYPES.Identifier && specifier.imported.name === 'v4') {
-          updatedUuid4Alias = specifier.local.name;
-        }
-        break;
-      case AST_NODE_TYPES.ImportDefaultSpecifier:
-        updatedUuidDefaultAlias = specifier.local.name;
-        break;
+    if (specifier.type === AST_NODE_TYPES.ImportSpecifier) {
+      if (
+        node.source.value === 'uuid' &&
+        specifier.imported.type === AST_NODE_TYPES.Identifier &&
+        specifier.imported.name === 'v4'
+      ) {
+        aliases.uuid4Alias = specifier.local.name;
+      } else if (
+        node.source.value === 'node:crypto' &&
+        specifier.imported.type === AST_NODE_TYPES.Identifier &&
+        specifier.imported.name === 'randomUUID'
+      ) {
+        aliases.cryptoRandomUUIDAlias = specifier.local.name;
```

Comment:
should cryptoRandomUUIDAlias be renamed as sth like nodeCryptoRandomUUIDAlias so that we can differenciate it with JS's built-in crypto?